data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaSourceConfig.java

```diff
@@ -30,19 +31,58 @@ public class KafkaSourceConfig {
   @Size(min = 1, max = 10, message = "The number of Topics should be between 1 and 10")
   private List<TopicsConfig> topics;
 
+  @JsonProperty("auth_type")
+  @NotNull
+  @Valid
+  private String authType;
+
+  @JsonProperty("schema")
+  @NotNull
+  @Valid
+  private SchemaConfig schemaConfig;
+
+  @JsonProperty("schema")
```

Comment:
I guess this is a typo? should be something like "authConfig" ?

Reply:
typo removed and plain text auth type added